### PR TITLE
1263 - Update Triggers and Flow for Kanbanize Card movement WRT Pull …

### PR DESCRIPTION
**Pull request by martindevnow**
…Requests

### CODE REVIEW

File: .github/actions/kanbanize/README.md [hunk 1]
```diff
@@ -5,3 +5,68 @@
...
+eventData.context.payload.pull_request.head.ref === 'infra/1234-branch-name';
+eventData.eventName === 'pull_request';
+eventData.headRef === 'infra/1234-branch-name';
+```
+
+### When a user opens a PROPER PR, move the ticket to "Ready for Code Review"
+
+```
+eventData.context.payload.pull_request.base.ref === 'main';
+eventData.context.payload.pull_request.draft === true
```
Comment (vighfx) [inline .github/actions/kanbanize/README.md:25]: Should this be false for `Ready for Code Review`?

**Review by vighfx — APPROVED** [review]
Looks good, minor comment / clarification on README

File: .github/actions/kanbanize/README.md (hunk 1 above)
Comment (martindevnow) [reply .github/actions/kanbanize/README.md:25]: Good catch. Luckily I caught this when translating these into code, but I will update here as well. Thanks

**Review by vighfx — APPROVED** [review]
Looks great, thanks for the updates on these

**Review by crajapakshe — APPROVED** [review]
LGTM

**Pull request merged**